tools/eval/README.md

```diff
@@ -19,6 +19,7 @@ Supported datasets:
 - [ImageNet](#imagenet)
 - [WIDERFace](#widerface)
 - [LFW](#lfw)
+- [ICDAR](#icdar)
```

Comment:
If the current code is compatible with different versions of ICDAR dataset, we can add comments in the end.